apps/src/code-studio/pd/permission.js

```diff
@@ -0,0 +1,12 @@
+export default class Permission {
+  constructor() {
+    this.isAdmin = this.hasPermission('workshop_admin');
```

Comment:
Please, please, please don't conflate workshop_admin with admin. These are very distinct concepts on the backend.